### ISSUE TITLE
compare_scholar_careers bug?

### ISSUE THREAD

**Issue by nail2001** [jkeirstead/scholar#36]
If I execute this code:
```
ids <- c('dYWNWicAAAAJ', 'R2ZrHtsAAAAJ', 'yg0LY3QAAAAJ', 'OixfQOcAAAAJ', 'k6b-lLYAAAAJ', 'g-U6tyoAAAAJ', 'ixGcu5gAAAAJ', 'docGNYEAAAAJ', 'kU2jvOMAAAAJ', '_XJMeP0AAAAJ', 'aqsaHZwAAAAJ', 'wEU99lsAAAAJ')
cmp2 <- compare_scholar_careers(ids)

```
I get: 

Error in data.frame(year = years, cites = vals) :
  arguments imply differing number of rows: 7, 6
Calls: compare_scholar_careers ... lapply -> FUN -> cbind -> get_citation_history -> data.frame
Execution halted

PS. these same instructions it worked before. but I guess some changes in the google data triggers this error.

**Comment by mrmvergeer** [comment]
I also had issues with compare_scholar_careers. It worked when I limited the number of IDs to 7. Maybe Google limited the number of simultaneous calls.

**Comment by mrmvergeer** [comment]
I now get this error: 
> Error in get_resp(url, attempts_left - 1) : 
  Cannot connect to Google Scholar. Is the ID you provided correct?>

**Comment by TheVandalyst** [comment]
did anyone found out a solution  ?

**Comment by Shicheng-Guo** [comment]
I also met such problem frequently. I think it is 'google scholar' problem. It looks google scholar always try to prohibit such kind of operation.

**Comment by mrmvergeer** [comment]
I just tried it with 15 IDs and I got no errors. 
However, repeatedly calling this procedure  produces the error. 
So, it's best to pause before to rerun this code. How long? I know some other calls to Google Scholar need a pause of about 15 minutes

**Comment by Shicheng-Guo** [comment]
Agree. It is a good idea to save the retrieved google scholar profiles to
avoid frequently calling.
